RELEASES.md

```diff
@@ -1,3 +1,98 @@
+Version 1.76.0 (2024-02-08)
+==========================
+
+<a id="1.76.0-Language"></a>
+
+Language
+--------
+- [Document Rust ABI compatibility between various types](https://github.com/rust-lang/rust/pull/115476/)
+- [Also: guarantee that char and u32 are ABI-compatible](https://github.com/rust-lang/rust/pull/118032/)
+- [Warn against ambiguous wide pointer comparisons](https://github.com/rust-lang/rust/pull/117758/)
+
+<a id="1.76.0-Compiler"></a>
+
+Compiler
+--------
+- [Lint pinned `#[must_use]` pointers (in particular, `Box<T>` where `T` is `#[must_use]`) in `unused_must_use`.](https://github.com/rust-lang/rust/pull/118054/)
+- [Soundness fix: fix computing the offset of an unsized field in a packed struct](https://github.com/rust-lang/rust/pull/118540/)
+- [Soundness fix: fix dynamic size/align computation logic for packed types with dyn Trait tail](https://github.com/rust-lang/rust/pull/118538/)
+- [Add arm64e-apple-ios & arm64e-apple-darwin targets](https://github.com/rust-lang/rust/pull/115526/)
+- [Add `$message_type` field to distinguish json diagnostic outputs](https://github.com/rust-lang/rust/pull/115691/)
+- [Enable Rust to use the EHCont security feature of Windows](https://github.com/rust-lang/rust/pull/118013/)
+- [Add new targets {x86_64,i686}-win7-windows-msvc](https://github.com/rust-lang/rust/pull/118150/)
+- [Add new tier 3 aarch64-apple-watchos target](https://github.com/rust-lang/rust/pull/119074/)
+
+<a id="1.76.0-Libraries"></a>
+
+Libraries
+---------
+- [adds a column number to `dbg!()`](https://github.com/rust-lang/rust/pull/114962/)
+- [Add `std::hash::{DefaultHasher, RandomState}` exports](https://github.com/rust-lang/rust/pull/115694/)
+- [Fix rounding issue with exponents in fmt](https://github.com/rust-lang/rust/pull/116301/)
+- [Stablize arc_unwrap_or_clone](https://github.com/rust-lang/rust/pull/116949/)
+- [Add T: ?Sized to `RwLockReadGuard` and `RwLockWriteGuard`'s Debug impls.](https://github.com/rust-lang/rust/pull/117138/)
+- [Stabilize `ptr::{from_ref, from_mut}`](https://github.com/rust-lang/rust/pull/117824/)
+- [Stabilize RFC3324 dyn upcasting coercion](https://github.com/rust-lang/rust/pull/118133/)
```

Comment:
This was stabilized by a T-lang [fcp](https://github.com/rust-lang/rust/issues/65991#issuecomment-1670127912); should probably be moved into the Language section.

Reply:
Anyway the stsbilization has been reverted now.